app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ViewProfileScreenTest.kt

```diff
@@ -365,9 +374,7 @@ class ViewProfileScreenTest {
             ParkingRackType.TWO_TIER,
             ParkingProtection.COVERED,
             0.0,
-            true,
-            owner = TestInstancesUser.user1.public.userId,
```

Comment:
Are you sure you should be removing those arguments ?

Reply:
I just did a bad rebase accidentally and it's not the only place where it's broken. No this is not norma. Yes i am currently fixing it.

Reply:
No problem don't forget to ping me if you need help or if you finished rebasing so I can check :+1: